### ISSUE TITLE
Cannot create vmagent due to service account not created

### ISSUE THREAD

**Issue by nthienan** [VictoriaMetrics/operator#539]
I failed to deploy a new vmagent due to an error:
```
 Error creating: pods "vmagent-xxx-d8668bd86-" is forbidden: error looking up service account aa/vmagent-xxx: serviceaccount "vmagent-xxx" not found
```
I checked service account. There is no service account name `vmagent-xxx` created.

I use chart `victoria-metrics-operator`, version `0.14.2`, `vmoperator` version: `v0.28.4`

**Comment by f41gh7** [comment]
This issue was fixed at v0.28.5 operator version, you have to update helm chart version to the `0.14.3` version.

**Comment by nthienan** [comment]
working after upgrading to v0.28.5
